Jenkinsfile_CNP

```diff
@@ -39,7 +39,7 @@ withPipeline(type, product, component) {
     enableDockerBuild()
     installCharts()
 
-    env.S2S_URL_FOR_TESTS = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
```

Comment:
Should be aat! We need to be VERY CAREFUL with this. I am going to correct it on your behalf this time.